assertj-core/src/main/java/org/assertj/core/error/KeyStoreShouldContainAlias.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * Copyright 2012-2023 the original author or authors.
+ */
+package org.assertj.core.error;
+
+public class KeyStoreShouldContainAlias extends BasicErrorMessageFactory {
+
+  public KeyStoreShouldContainAlias(String alias) {
+    super("Expected KeyStore to contain alias <%s>, but it did not", alias);
+  }
+
+  public static KeyStoreShouldContainAlias shouldContainAlias(String alias) {
```

Comment:
rename to `keyStoreShouldContainAlias` to avoid any ambiguity as from where the alias comes from

Reply:
Done